diff --git a/qtbase/src/plugins/platforms/windows/qwindowsmousehandler.cpp b/qtbase/src/plugins/platforms/windows/qwindowsmousehandler.cpp
index 5f34c79..9cb9bce 100644
--- a/qtbase/src/plugins/platforms/windows/qwindowsmousehandler.cpp
+++ b/qtbase/src/plugins/platforms/windows/qwindowsmousehandler.cpp
@@ -61,7 +61,7 @@
  * versions. */
 
-#if defined(Q_CC_MINGW) || !defined(TOUCHEVENTF_MOVE)
+#if !defined(TOUCHEVENTF_MOVE)
 
 typedef struct tagTOUCHINPUT {
     LONG x;
